libs/payments/events/src/lib/emitter.factories.ts

```diff
@@ -29,7 +30,7 @@ export const SubscriptionEndedFactory = (
   priceInterval: faker.helpers.arrayElement(['month', 'year']),
   priceIntervalCount: faker.number.int({ min: 1, max: 12 }),
   providerEventId: faker.string.uuid(),
-  voluntaryCancellation: faker.datatype.boolean(),
+  voluntaryCancellation: faker.helpers.enumValue(CancellationReason),
```

Comment:
Should this name be changed from `voluntaryCancellation` to `cancellationReason`?